config/crd/v1/bases/velero.io_podvolumerestores.yaml

```diff
@@ -116,6 +116,10 @@ spec:
               snapshotID:
                 description: SnapshotID is the ID of the volume snapshot to be restored.
                 type: string
+              sourceNamespace:
```

Comment:
Rather than defining the `sourceNamespace` here to refer to the backup repository implicitly, I prefer to define the backup repository name directly.

PVR controller doesn't need to know how to assemble the backup repository name, this is the 
responsibility of PVR creator not the PVR controller. 

PVR controller only needs to know the backup repository name

Reply:
@ywk253100 If the `PVB` is synced from storage in a new cluster, once first connecting a repo that BackupRepository CR does not exist, RepositoryEnsurer will create a new one, and the BackupRepository will be changed, so if restoring the BackupRepository name could not found the related repo

Reply:
In the migration use case, the new BackupRepository will be created on the new cluster and the newly created PVRs should use the new BackupRepository name, so the issue you mentioned should not be there.

But I'm OK if you insist on this design as we discussed offline. I'm resolving the comment.